cromwell.examples.conf

```diff
@@ -277,13 +277,13 @@ backend {
         # `temporary-directory` creates the temporary directory for commands.
         #
         # If this value is not set explicitly, the default value creates a unique temporary directory, equivalent to:
-        # temporary-directory = "mktemp -d \"$PWD\"/tmp.XXXXXX"
+        # temporary-directory = "$(mktemp -d \"$PWD\"/tmp.XXXXXX)"
         #
         # The expression is run from the execution directory for the script. The expression must create the directory
         # if it does not exist, and then return the full path to the directory.
         #
         # To create and return a non-random temporary directory, use something like:
-        # temporary-directory = "mkdir -p /tmp/mydir && echo /tmp/mydir"
+        # temporary-directory = "$(mkdir -p /tmp/mydir && echo /tmp/mydir)"
```

Comment:
ToL is it not possible for Cromwell to just auto-wrap these values as it reads them in? Right now this just feels like a weird thing you have to to remember to do (differently from before) in the config file